app/assets/v2/css/submit_bounty.css

```diff
@@ -227,3 +227,13 @@ input:read-only {
 .letter-spacing {
   letter-spacing: 0.1em;
 }
+
+.releaseAfterFormGroup {
```

Comment:
Please lets follow the class name convention not camelcase

Reply:
After that this will be good for merge @vince0656

Reply:
ok no problem, I'll make that change and push it up